app/build.gradle.kts

```diff
@@ -51,6 +55,15 @@ dependencies {
     implementation(Libs.Android.Navigation.UI)
     implementation(Libs.Android.Lifecycle.LIVE_DATA)
     implementation(Libs.Android.Lifecycle.VIEW_MODEL)
+    implementation(Libs.Android.COMPOSE.ACTIVITY)
+    implementation(Libs.Android.COMPOSE.MATERIAL)
+    implementation(Libs.Android.COMPOSE.MATERIAL_THEME)
+    implementation(Libs.Android.COMPOSE.ANIMATION)
+    implementation(Libs.Android.COMPOSE.UI_TOOL)
+    implementation(Libs.Android.COMPOSE.VIEW_MODEL)
+
+    //GSON
+    implementation(Libs.GSON.GSON)
```

Comment:
can we use Moshi? which is the replacement of GSON. https://github.com/square/moshi